doc/rst/developer/bestPractices/ContributorInfo.rst

```diff
@@ -102,91 +319,92 @@ Submit pull request
-writing or modifying tests.
+Find a reviewer
++++++++++++++++
 
-* Testing your patch:
+* When you believe your pull request is ready for review, send it to a member of
+  the core Chapel team (or to the chapel-developers_ list if there isn't an
+  obvious person to review it). To make it easy to spot, please include `[PR]`
+  prior to the subject of your email.  Even the developers that have write
+  access to the Chapel repository need to have all non-trivial changes
+  reviewed. Developers who have been given write access can merge trivial
+  changes (e.g. small bug fixes, documentation changes) without review.
 
-  * Any addition/change to the Chapel test system should pass testing when that
-    test/directory is run with ``start_test`` (and performance tests should
-    also pass testing for ``start_test -performance``).
+  Note: Ideally, someone should volunteer to review your pull request within a
+  day or two. If this doesn't happen, feel free to make some noise. Ideally the
+  review should take place within a few days, though timing may vary depending
+  on other deadlines.
 
-  * Changes to the Chapel implementation should not cause
-    regressions. Developers are responsible for doing a degree of testing
-    that's appropriate for their change (see guidance below) and then can rely
-    on nightly regression testing to worry about the full cross-product of
-    configurations.
+* See `Reviewer responsibilities`_ for details on what performing a review on
+  another contributor's code entails.
 
-    * At a minimum, patches should pass correctness testing for the full test/
-      directory hierarchy for:
+.. _Work with your reviewers:
 
-      * ``CHPL_*_PLATFORM=linux64``
-      * ``CHPL_*_COMPILER=gnu``
-      * ``CHPL_COMM=none``
-      * ``CHPL_TASKS=<default>``
+Work with your reviewers
+++++++++++++++++++++++++
 
-    * Most developers will start by focusing on a subdirectory of tests that
-      exercise the features they changed, or test/release/ as a suite of tests
-      that exercises a rich and important slice of the language.
-
-    * Changes that are likely to affect multi-locale executions should also be
-      tested against tests that exercise multi-locale capabilities with
-      ``CHPL_COMM=gasnet``.  A common subset is: ``test/release/``,
-      ``test/multilocale/``, and ``test/distributions/``.
-
-    * Changes that are likely to cause portability issues should be tested
-      against different platforms and compilers to avoid fallout in the nightly
-      testing to the extent possible.
+* Iterate with the reviewer until you're both satisfied. If you can't come to
+  agreement, one of you should bring other developers (individually or via
+  chapel-developers_) into the conversation to get a broader opinion (one of
+  the jobs of the reviewer is to serve as a proxy for other developers, or to
+  bring those developers into the conversation if they feel unqualified to do
+  so).
```

Comment:
This seems pretty good. However I think it could point out two more things:
 * one should generally try to do whatever their reviewer asks
 * sometimes a reviewer will ask for something really hard. Just try to make sure they understand the magnitude of the request & try to discuss if it's really necessary to do before merging.